spec/requests/calculators_spec.rb

```diff
@@ -87,6 +93,24 @@
   end
 
   describe "POST /receive_recomendations" do
```

Comment:
remove there recieve_recomendations tests, they are useless, this feature is not available at the moment